docs/conf.py

```diff
@@ -15,7 +15,7 @@
 
 # sys.path.insert(0, os.path.abspath("."))
 # sys.path.insert(0, os.path.abspath(".."))
-sys.path.insert(0, os.path.abspath("../.."))
+  sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
It looks like this still hasn't been reverted back. There's still an extra space here:

```suggestion
sys.path.insert(0, os.path.abspath("../.."))
```